boards/modalai/voxl2/target/voxl-px4-hitl

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+CONFIG_FILE="/etc/modalai/voxl-px4-hitl.conf"
+
+DAEMON_MODE=DISABLE
+
+# Try to source configuration variables from a file.
+# This will override anything already set in the environment.
+if [ -f "$CONFIG_FILE" ]; then
+    echo "[INFO] Reading from $CONFIG_FILE"
+    source $CONFIG_FILE
+else
+    echo "[INFO] $CONFIG_FILE not found, using defaults"
+fi
```

Comment:
I think you can just remove lines 3 through 14. Skip the configuration file since there is nothing in it that you need.